src/theme/index.scss

```diff
@@ -1,6 +1,7 @@
 @import "base/variables";
 @import "base/elements";
 @import "base/mediaqueries";
+//@import "base/fonts";
```

Comment:
the fonts file is not being imported here. In terms of performance, having the fonts inline (in the html file) makes sense to me, but in terms of code standards I like the idea of having a separate file and import it here.

Reply:
See the results of my benchmarks, critical CSS prevents FOIT and FOUT and is often better regarding performance (results). This line here and the file are open for discussion and are the alternative approach (but trigger FOIT and worse performance).

Reply:
I'd say we go for performance, thoughts?